source/expressions.tex

```diff
@@ -4542,9 +4539,11 @@
 the \grammarterm{new-initializer}~(\ref{dcl.init.aggr}, \ref{dcl.init.string}).
 
 \pnum
+\indextext{\idxcode{new}}%
+If the \grammarterm{expression} in a \grammarterm{noptr-new-declarator}
+is present, it is implicitly converted to \tcode{std::size_t}.
 \indextext{function!allocation}%
```

Comment:
Do you think this index should precede the new text?

Reply:
It's existing stuff, and I'm not too happy about the indexing here in general. I'd like to leave it as-is for now.